gem/tests/test_views.py

```diff
@@ -431,6 +440,23 @@ def test_comment_filters(self):
 
         self.assertFalse(comment_form.is_valid())
 
+    def test_anonymous_comment_translation(self):
```

Comment:
we don't need to test for pootle translations as they often get changed. so I don't think this is necessarily